docs/user-guide/create-page.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar-position: 3
+---
+
+# Create page
+
+This is where the actual magic happens and where you cook your resume with our
+drag-and-drop easy-to-use technology, along with multiple customizations.
+
+> The create panel is the place where you get to edit the details inside your resume.
+
+It gives you **editing options on the left** and an **A4 size preview on the right**.
```

Comment:
Lets write preview on right instead of A4 size preview on right because Its not A4 size

Reply:
It still says A4 size

Reply:
changed to "Live resume preview on right" 
is that change alright?

Reply:
Works